hm_pyhelper/miner_param.py

```diff
@@ -115,8 +115,21 @@ def get_gateway_mfr_version() -> Version:
 def get_ecc_location() -> str:
     ecc_list = get_variant_attribute(os.getenv('VARIANT'), 'SWARM_KEY_URI')
 
+    try:
+        with open("eccfile", 'r') as data:
```

Comment:
no value in it - i just don't know how to use the functools cache method. do you have a reference for what you are suggesting so I can take a look?

alternatively, i guess we could write it to persistent storage and then it would only need to run once on first boot (or if the device is purged). This actually may be a better way to handle it?

Reply:
Ok discussed on Slack and updated... https://nebraltd.slack.com/archives/C024BNQ1Y6T/p1679944794789009?thread_ts=1679944794.789009&cid=C024BNQ1Y6T